Makefile

```diff
@@ -11,6 +11,7 @@ include build/scripts/testing.mk
 include contracts/Makefile 
 include kurtosis/Makefile
```

Comment:
Tried run beacon-kit docker image from GH, it is giving error. Seems we don't have an image for my architecture.
`WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
Caught SIGILL in blst_cgo_init, consult <blst>/bindings/go/README.md.`
[5:03](https://berachain.slack.com/archives/D06RURL8C5T/p1723203183421119)
For the time being, I built one on my machine to unblock myself.

I am encountering below error, I am not passing RPC URL as should be picked from default i.e. http://localhost:8551 .
`2024/08/09 11:27:35 ERROR startup failure error="failed to unmarshal app config: 1 error(s) decoding:\n\n* 'beacon-kit.engine.rpc-dial-url' expected a map, got 'string'"`

Reply:
figured it out.